### PR TITLE
add llm

### PR DESCRIPTION
**Pull request by junjiejiangjjj**
#37

### CODE REVIEW

File: gptcache/adapter/langchain_llms.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+from typing import Optional, List
+from .adapter import adapt
+
+
+class LangChinLLMs:
```
Comment (SimFG) [inline gptcache/adapter/langchain_llms.py:5]: type error, `LangChain`

File: gptcache/adapter/langchain_llms.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    _llm = None
+    def __init__(self, llm):
+        if self._llm is not None:
+            raise RuntimeError('llm has been created')
```
Comment (SimFG) [inline gptcache/adapter/langchain_llms.py:9]: When Will the __init__ method be called except when the object is created?

Comment (junjiejiangjjj) [reply gptcache/adapter/langchain_llms.py:9]: Just to prevent users from creating duplicate

File: gptcache/adapter/langchain_llms.py (hunk 1 above)
Comment (junjiejiangjjj) [reply gptcache/adapter/langchain_llms.py:5]: did you mean rename `LangChinLLMs ` to `LangChain`

File: gptcache/adapter/langchain_llms.py (hunk 2 above)
Comment (junjiejiangjjj) [reply gptcache/adapter/langchain_llms.py]: fixed

**Pull request merged**